mlir/include/mlir/Dialect/Affine/Analysis/Utils.h

```diff
@@ -610,6 +610,14 @@ FailureOr<AffineValueMap>
 simplifyConstrainedMinMaxOp(Operation *op,
                             FlatAffineValueConstraints constraints);
 
+/// Find the innermost common `Block` of `a` and `b` in the affine scope
+/// that `a` and `b` are part of. Return nullptr if they belong to different
+/// affine scopes. Also, return null if they do not have a common `Block`
```

Comment:
Nit: null -> nullptr

Reply:
Nit: Should `Block`, `then` and `else` be inside backticks? They are not function arguments.

Reply:
It's not for just function arguments, but I use it for other proper MLIR notation that would otherwise make the sentence broken.

Reply:
nIT: Also, return **`nullptr`** if they do not have a common `Block`